Delicate-Butterflies/Bangazon-API
# Branch1 ask

Related Issue: #72 

Changes: 
added user and product table create functionality
serializing drop, create, insert into product tables

Any other Testing Steps:
- check for the attributes with the one we decided on
- check for the function syntax and modularity of table/product.js
